source/index.markdown

```diff
@@ -27,8 +27,7 @@ These pages gives an overview of how to use Elastica. You can find the complete
 
 Compatibility
 --------------
-Elastica is tested with PHP 5.4 and later. Most functionality still works with PHP 5.3 but future support is not guaranteed. It Versions prior and equal to v0.19.8.0 are compatible with PHP 5.2
-
+Elastica is tested with PHP 7.0 and later. It Versions prior and equal to v0.19.8.0 are compatible with PHP 5.2
```

Comment:
Let's just go with the first sentence but saying something like: `Elastica 6.x is tested with ...`. The second sentence by now will cause probably more confusion then solve  it.

Reply:
Done :+1: